scripts/release_mp.py

```diff
@@ -311,9 +315,9 @@ def main():
 
     try:
         usage = ("""usage: %prog <command> [options]
-       %prog clone [-r alias] [-m URL] [-d PATH]
-       %prog branch [-r alias] [-m URL] [-d PATH] [--rf package] [--dryrun]
-       %prog prepare [-r alias] [-m URL] [-d PATH] [--rf package] [--dryrun]
+       %prog clone [-r alias] [-m URL] [-d PATH] [--depth N] [--unshallow]
+       %prog branch [-r alias] [-m URL] [-d PATH] [--depth N] [--unshallow] [--rf package] [--dryrun]
+       %prog prepare [-r alias] [-m URL] [-d PATH] [--depth N] [--unshallow] [--rf package] [--dryrun]
```

Comment:
These are Git low level behaviors, I'm not comfortable with putting them here, with this naming.

Reply:
ok, any suggestion ?

Reply:
I've been thinking about this: what would you think of using environment variables then ?
Something like:
`env GIT_CLONE_DEPTH=1 python clone.py master` and `env GIT_UNSHALLOW=true python clone.py master` ?

Reply:
Why not using a file? It would be generic and allow to tune any Git behavior.

       -f config-file, --file config-file
           Use the given config file instead of the one specified by GIT_CONFIG.

       --blob blob
           Similar to --file but use the given blob instead of a file. E.g. you can use master:.gitmodules to read values from the file .gitmodules in the master branch. See "SPECIFYING REVISIONS"
           section in gitrevisions(7) for a more complete list of ways to spell blob names.

Reply:
A file is not practical, these options will have to be passed to scripts calling ./clone.py (like for instance QA scripts). Command-line options as suggested are perfectly fine.